sdks/java/core/src/main/java/org/apache/beam/sdk/util/PubsubClient.java

```diff
@@ -387,10 +393,10 @@ public int hashCode() {
   }
 
   /**
-   * Gracefully close the underlying transport.
+   * @inheritDoc
    */
   @Override
-  public abstract void close();
+  public abstract void close() throws IOException;
```

Comment:
Sorry -- I may have asked this already -- but why include L394–L399 at all? This abstract class implements `Closeable` should implicitly do exactly this.